yarn.lock

```diff
@@ -9,6 +9,21 @@
   dependencies:
     "@babel/runtime" "^7.11.2"
 
+"@adeira/sx@^0.15.0":
```

Comment:
Some package doesn't use the latest `@adeira/sx` version? This looks wrong. 🤔

Reply:
Yeah, this came when rebasing master into this branch. It should be fixed now.